packages/conventional-recommended-bump/README.md

```diff
@@ -97,6 +97,11 @@ Type: `string` Possible values: `'major'`, `'minor'` and `'patch'`
 
 The value of what it should release as.
 
+##### tagPrefix
```

Comment:
I think this section needs to be moved up, underneath the `#### options` section, instead of the `#### parserOpts` section. In the code below the option is pulled from `options` and not `parserOpts`.